toc-bar.user.js

```diff
@@ -310,6 +371,7 @@
 
   position: fixed;
   z-index: 9000;
+  font-family: 'Roboto';
```

Comment:
I'm not sure if this should be added, or maybe it's not enough to add just one font-family